tutorials/mongoose/deleteMany.md

```diff
@@ -0,0 +1,41 @@
+If you wish to delete several documents in mongoose, use the `deleteMany()` function.
```

Comment:
"The `deleteMany()` function is how you can delete multiple documents from a collection using Mongoose."